repositories/MerchOrderRepository.ts

```diff
@@ -48,7 +49,18 @@ export class MerchOrderRepository extends BaseRepository<OrderModel> {
    * Gets all orders for a given user. Returns the order joined with its pickup event and user.
    */
   public async getAllOrdersForUser(user: UserModel): Promise<OrderModel[]> {
-    return this.repository.find({ user });
```

Comment:
@dowhep The comment says this query joins with pickup event and user but I was confused where it does this since the join isn't explicit. I rewrote the query to match the query on line 58 (`getAllOrderswithItemsForUser`), but I'm just wondering where the join was done in the first place